docs/cep/tutorials/using_rest_api.md

```diff
@@ -178,234 +178,232 @@ print("\nStream App Deleted: ", result)
-
-        const producer = new WebSocket(producerUrl);
-
-        producer.onopen = function () {
-          console.log("WebSocket:Producer is open now for " + streamName);
-        };
-
-        producer.onerror = function () {
-          console.log(
-            "Failed to establish WebSocket:Producer connection for " + streamName
-          );
-        };
-
-        producer.onclose = function () {
-          console.log("Closed WebSocket:Producer connection for " + streamName);
-        };
-
-        producer.onmessage = function () {
-          console.log("WebSocket:Producer message sent successfully");
-        };
-
-        await new Promise((resolve) => setTimeout(resolve, 10000));
-
-        const INPUT_DATA = [
-          {
-            deviceID: "AD11",
-            roomNo: 200,
-            temperature: 18,
-          },
-          { deviceID: "AD11", roomNo: 201, temperature: 47 },
-        ];
-
-        producer.send(
-          JSON.stringify({
-            payload: btoa(JSON.stringify(INPUT_DATA[0])),
-          })
-        );
-
-        await new Promise((resolve) => setTimeout(resolve, 10000));
-
-        producer.send(
-          JSON.stringify({
-            payload: btoa(JSON.stringify(INPUT_DATA[1])),
-          })
-        );
-
-        await new Promise((resolve) => setTimeout(resolve, 10000));
-
-        producer.close();
-
-        /* ----------------------------- Verify results ----------------------------- */
-
-        const SELECT_QUERY = "FOR doc IN tutorialAppOutputTable return doc";
-
-        const result = await connection.req(`/_fabric/_system/_api/cursor`, {
-          body: {
-            id: "tutorialStreamAppQuery",
-            query: SELECT_QUERY,
-            bindVars: {},
-          },
-          method: "POST",
-        });
-
-        console.log("INPUT SENT --->", INPUT_DATA);
-        console.log("OUTPUT DATA --->", result.results);
-
-        /* ---------------------------- Delete StreamApp ---------------------------- */
-        const deletion = await connection.req(
-          `/_fabric/_system/_api/streamapps/${STREAM_APP_NAME}`,
-          {
-            method: "DELETE",
-          }
-        );
-
-        console.log("STREAM APP DELETED SUCCESSFULLY", deletion);
-      } catch (e) {
-        console.error(e);
+    await connection.req(
+      `/_fabric/_system/_api/streamapps/${streamAppName}/active?active=true`,
+      {
+        method: "PATCH"
       }
+    );
+
+    console.log("Activating stream app...", streamAppName);
+
+    await new Promise((resolve) => setTimeout(resolve, 10000));
+
+    console.log("Stream app activated successfully");
+
+    /* ------------------ Publish messages to Sample StreamApp ------------------ */
+    const region = isGlobal ? "c8global" : "c8local";
+    const streamName = `${region}s.${stream}`;
+
+    // Fetching local URL in case the stream is local (which is defined in the streamapp)
```

Comment:
And here is a third spelling of "streamapp"

Reply:
This one I missed yeah.. I will change it

Reply:
Thanks for the reviews